packages/get-platform/src/getPlatform.ts

```diff
@@ -288,6 +288,11 @@ type ComputeLibSSLSpecificPathsParams = {
 }
 
 export function computeLibSSLSpecificPaths(args: ComputeLibSSLSpecificPathsParams) {
+  if (process.env.LD_LIBRARY_PATH) {
+    debug(`Found 'LD_LIBRARY_PATH' specified as env-var, ignoring platform specific lib-dir paths...`)
+    return process.env.LD_LIBRARY_PATH.split(':')
```

Comment:
While `LD_LIBRARY_PATH` does take precedence, it does not completely supplant the paths from `ld.so.conf`, so we still have to check the standard paths.

Consider the case when `LD_LIBRARY_PATH` is required for native dependencies of some other Node.js addon that an application uses (a real example from my experience is a proprietary library for interfacing with certain devices) but OpenSSL is installed via the package manager.

Reply:
Should be fixed in #20381 :)